include/mongoose_metrics_definitions.hrl

```diff
@@ -57,3 +57,10 @@
                            {send_pend,0}
                           ]).
 
+-define(INET_STATS_METRICS, #{recv_oct => spiral,
```

Comment:
I wanted to put it near `INET_STATS`, as this is what `mongoose_metrics` uses to get the values. I guess I just used the convention that's in the header file. I can change it to a function if it's better.
Or do you think it could go somewhere else in the code entirely?